### PR TITLE
[2.x] Add `expectsDatabaseQueryCount` to Database

### PR DESCRIPTION
**Pull request by milwad-dev**
If accepted I added tests

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes

### CODE REVIEW

File: src/Database.php [hunk 1]
```diff
@@ -118,3 +118,15 @@ function seed($class = 'Database\\Seeders\\DatabaseSeeder')
...
 }
+
+/**
+ * Specify the number of database queries that should occur throughout the test.
+ *
+ * @param  int  $excepted
+ * @param  $connection
+ * @return TestCase
+ */
+function expectsDatabaseQueryCount(int $excepted, $connection = null)
```
Comment (nunomaduro) [inline src/Database.php]: Seems that we are missing a type here?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/Database.php (hunk 1 above)
Comment (milwad-dev) [reply src/Database.php]: Added.

**Comment by milwad-dev** [comment]
@nunomaduro

File: src/Database.php [hunk 2]
```diff
@@ -118,3 +118,15 @@ function seed($class = 'Database\\Seeders\\DatabaseSeeder')
...
 }
+
+/**
+ * Specify the number of database queries that should occur throughout the test.
+ *
+ * @param  int  $excepted
+ * @param  string|null  $connection
+ * @return TestCase
+ */
+function expectsDatabaseQueryCount(int $excepted, $connection = null)
```
Comment (nunomaduro) [inline src/Database.php]: Can you add type hints to the connection argument?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/Database.php (hunk 2 above)
Comment (milwad-dev) [reply src/Database.php]: Fixed @nunomaduro

**Pull request merged**